force-app/main/default/classes/AFFL_MultiRecordTypeMapper.cls

```diff
@@ -336,29 +319,30 @@ public virtual with sharing class AFFL_MultiRecordTypeMapper {
      * @param accountRecordType The name of the Account record type as entered in the Affiliation Mappings.
```

Comment:
Might wanna update the comments here? looks like the param at some point in time was supposed to be the account record type but now it looks like we are passing in an affiliation.